content/en/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

```diff
@@ -14,9 +15,29 @@ in Kubernetes 1.20, there were questions on how this will affect various workloa
 installations. Our [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq/) is there to help you
 to understand the problem better.
 
-It is recommended to migrate from dockershim to alternative container runtimes.
-Check out [container runtimes](/docs/setup/production-environment/container-runtimes/)
-section to know your options. Make sure to
-[report issues](https://github.com/kubernetes/kubernetes/issues) you encountered
-with the migration. So the issue can be fixed in a timely manner and your cluster would be
-ready for dockershim removal.
+<!-- body -->
+
+If you use Docker via dockershim as your container runtime, the Kubernetes project
+recommends that you migrate to an alternative container runtime.
+If you're not sure whether you are using Docker,
+[find out what container runtime is used on a node](/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/).
```

Comment:
If you're not sure whether you are using Docker, see ....

Reply:
I like the wording:
> If you're not sure whether you are using Docker, find out what container runtime is used on a node.

@kbhawkey can you highlight what about it isn't right?

Reply:
The text does not indicate to the user that they are moving to another doc or page section (no see [stuff]()).
I'm not sure where this text links?

Reply:
I'd argue that the link formatting would indicate that.

(I'd also maybe say "we recommend", idk "The Kubernetes project" sounds very imposing to me)

Reply:
We [avoid using we](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) in docs.